lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -217,6 +208,60 @@ def el8?
       def el_major_version
         return os_facts['os']['release']['major'] if el?
       end
+
+      def foreman_plugin_prefix
+        if el7?
+          'tfm-rubygem-foreman_'
```

Comment:
As mentioned in the proposal above, we have plugins that use `foreman-tasks` notation. And of course there is also katello, which doesn't use the foreman prefix at all.